lighthouse-core/gather/gatherers/link-elements.js

```diff
@@ -79,15 +81,24 @@ function getLinkElementsInDOM() {
 }
 /* c8 ignore stop */
 
-class LinkElements extends Gatherer {
+class LinkElements extends FRGatherer {
+  constructor() {
+    super();
+    /** @type {LH.Gatherer.GathererMeta<'DevtoolsLog'>} */
+    this.meta = {
+      supportedModes: ['timespan', 'navigation'],
+      dependencies: {DevtoolsLog: DevtoolsLog.symbol},
+    };
```

Comment:
If `meta` is defined outside the constructor, we get build errors on `yarn build-devtools`.

Reply:
did you run into https://github.com/GoogleChrome/lighthouse/issues/12134? :)

Reply:
Yes

Reply:
cool, add a comment linking to that issue to stop someone else from spending any time "fixing" it? :)